src/main/java/com/google/api/codegen/transformer/ruby/GeneratedLibraryViewModel.java

```diff
@@ -0,0 +1,55 @@
+/* Copyright 2016 Google Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.api.codegen.transformer.ruby;
+
+import com.google.api.codegen.transformer.ModelTypeTable;
+import com.google.api.codegen.transformer.SurfaceNamer;
+import com.google.api.tools.framework.model.Interface;
+import com.google.common.base.Preconditions;
+
+/**
+ * ViewModel for a generated library. This is used by Ruby for the moment, but there's nothing
+ * Ruby-specific about it.
+ */
+// TODO(jcanizales): Move it to/merge it with something common to all languages. Reuse it for Ruby
+// MVVM.
+public class GeneratedLibraryViewModel {
+  private final Interface service;
+  private final SurfaceNamer namer;
+  private final ModelTypeTable typeTable;
+
+  public GeneratedLibraryViewModel(
```

Comment:
These view classes do not follow the concept of MVVM, where view models are only data containers. We should not include any namers or type tables in the views, put this kind of logic into transformers instead.

Also this view class should not be necessary if you reuse `GapicSurfaceTestClassView`